### ISSUE TITLE
move __tests__ into src

### ISSUE THREAD

**Issue by christopher-johnson** [ProjectMirador/research-and-demos#83]
since the tests are source files, it would be correct to put them in the `src` directory.   Also, IDEA has an issue that if the tests are not in `src` than it bails before execution.   In Java, it is conventional to have the tests in a `test` directory and the tested code to be in `main` within the `src` directory.  This is a pretty solid pattern that might be worth emulating in JS.

**Comment by mejackreed** [comment]
Just historical context here, we put them in `./__tests__` because of the convention of the Jest community: https://jestjs.io/docs/en/configuration#testmatch-array-string .

**Comment by christopher-johnson** [comment]
right.  the double underscore convention can be preserved, though I find it to be a somewhat "tedious to type" semantic.  if the tests are named like `?(*.)(spec|test|unit)` then the `testMatch` pattern in the #82 config will also find them in `src/test`, `src/__tests__` or `src/something/test`.   btw, the IDEA question was local to my environment and has no bearing on this issue.
